components/nav/index.styl

```diff
@@ -0,0 +1,29 @@
+.navigation
+  display flex
```

Comment:
I am wondering about whether it'd be safer to do stuff like `table-cell` vertical centering rather than using Flexbox. Some Chinese browser-share Googling is probably in order.